src/Microsoft.OData.Core/ODataDeltaResourceSet.cs

```diff
@@ -4,32 +4,14 @@
 // </copyright>
 //---------------------------------------------------------------------
 
+using Microsoft.OData.Metadata;
+
 namespace Microsoft.OData
 {
     /// <summary>
     /// Describes a delta response.

```

Comment:
>delta response. [](start = 20, length = 15)

needs update?

Reply:
updated to "Describes a set of delta changes."

---
In reply to: [150157179](https://github.com/OData/odata.net/pull/989#discussion_r150157179) [](ancestors = 150157179)